qcengine/programs/torchani.py

```diff
@@ -110,9 +110,13 @@ def compute(self, input_data: 'ResultInput', config: 'JobConfig') -> 'Result':
             derivative = torch.autograd.grad(energy.sum(), coordinates)[0].squeeze()
             ret_data["return_result"] = np.asarray(derivative *
                                                    ureg.conversion_factor("angstrom", "bohr")).ravel().tolist()
+        elif input_data.driver == "hessian":
+            import torchani
```

Comment:
Could you move this import up to the `import torch` section on line 80 so that this is all imported in one area?

Reply:
Sure, fixed in 9939fab